src/components/scenes/CreateWalletReviewScene.js

```diff
@@ -43,7 +43,10 @@ class CreateWalletReviewComponent extends React.Component<Props, State> {
     Keyboard.dismiss()
   }
 
-  goToWalletList = () => Actions.popTo(WALLET_LIST_SCENE)
+  goToWalletList = () => {
```

Comment:
why not keep it `goToWalletList = () => this.props.navigation.navigate('walletListScene')`?
Is there a reason you don't want to return the value of navigate?